compiler-plugin/src/test/kotlin/arrow/meta/plugins/lens/LensTest.kt

```diff
@@ -27,12 +25,8 @@ class LensTest {
       """
 
     assertThis(CompilerTest(
-        config = {
-          addCompilerPlugins(compilerPlugin) + addDependencies(arrowAnnotations, arrowOptics)
-        },
-        code = {
-          codeSnippet.source
-        },
+        config = { metaDependencies + addDependencies(arrowOptics) },
```

Comment:
@raulraja I really like what you did with the dependencies bit here. I think I have to use that idea in my own branch!